.travis.yml

```diff
@@ -7,6 +7,16 @@ env:
 
 matrix:
   include:
+    # can't run tests on 1.31.0 due to rand dev-dependency
+    - rust: 1.31.0
+      os: linux
+      script
```

Comment:
You are missing a `:` here. This is probably why the Travis build isn't working.